integration/spark/src/main/scala/org/apache/carbondata/indexserver/IndexServer.scala

```diff
@@ -50,6 +51,7 @@ trait ServerInterface {
   /**
    * Used to prune and cache the index for the table.
    */
+  @throws(classOf[IOException])
```

Comment:
can ignore the testcase and revert this change

Reply:
no need to ignore testcase as external carbon segments are pruned by indexserver. non-carbon segments are not pruned. From `distributedPrunedRdd `when async calls to prune are made, `TableIndex.prune` method throws IOException and this is not handled in parent caller - `indexserver.getsplits`